### PR TITLE
Bind set_suffix and get_suffix in Tree

### PR DESCRIPTION
**Pull request by DarkMessiah**
Bind missing methods in Tree.

### CODE REVIEW

**Comment by DarkMessiah** [comment]
@Calinou Do I need to do something extra to pass all the checks? :)

**Comment by akien-mga** [comment]
Yes, the failed check shows that there is a style issue: https://travis-ci.org/github/godotengine/godot/jobs/683523938

It's not immediately visible on the diff but there's trailing whitespace on one line, which needs to be removed by amending the commit.

File: scene/gui/tree.cpp [hunk 1]
```diff
@@ -778,6 +778,9 @@ void TreeItem::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("set_text", "column", "text"), &TreeItem::set_text);
 	ClassDB::bind_method(D_METHOD("get_text", "column"), &TreeItem::get_text);
+
```
Comment (akien-mga) [inline scene/gui/tree.cpp:781]: Trailing whitespace here (even though GitHub doesn't display it for some reason - it used to).

**Comment by akien-mga** [comment]
As a side note, the email used to sign your commit (see header in https://patch-diff.githubusercontent.com/raw/godotengine/godot/pull/38495.patch ) seems not to be linked to your GitHub account, which is why the commit doesn't appear as authored by @DarkMessiah.

It's not a problem for Git or for us, but in case you want to reclaim ownership of that commit, you can add that email as secondary email in your GitHub settings.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.2.

**Pull request merged**